models/usersprintmapping.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
no need for this mapping as we are doing User --> WorkSpace mapping

Reply:
ok